File: Site/src/app/models/clienteModel.ts
import { enderecoModel } from "./enderecoModel";

export interface clienteModel {
    id: number;
    nome: string;
    dataDeNascimento: Date;
    tipoDeClienteSexo: number;
    dataDeCadastro: Date;
    dataDeAlteracao: Date | null;
    enderecos: enderecoModel[];
  }

File: Api/LoremIpsumLogistica.UI.Api/Models/ClienteModel.cs
using LoremIpsumLogistica.Dominio.Constantes;

namespace LoremIpsumLogistica.UI.Api.Models
{
    public class ClienteModel : EntidadeModel
    {
        public string Nome { get; set; }
        public DateTime DataDeNascimento { get; set; }
        public TipoDeClienteSexo TipoDeClienteSexo { get; set; }

    }
}
 